api/envoy/config/filter/http/cache/v2/cache.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.cache.v2;
+
+import "envoy/api/v2/route/route_components.proto";
+import "envoy/type/matcher/string.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.filter.http.cache.v2";
+option java_outer_classname = "CacheProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package = "envoy.extensions.filters.http.cache.v3";
+
+// [#protodoc-title: HTTP Cache Filter]
+// HTTP Cache Filter :ref:`overview <config_http_filters_cache>`.
+// [#extension: envoy.filters.http.cache]
```

Comment:
If you merge master to pick up the protodoc-title check, it will fail without a newline between this comment and the next message.

Reply:
done